CaPPMS/Shared/ProjectEntryAuthorizedDropdown.razor

```diff
@@ -210,14 +210,25 @@
             </tr>
 
             <div style="visibility:@gitHubErrorBar">
-                <b style="color:red">Could not create GitHub Repo... Please make sure Repo doesn't already exist!</b>
+                <p style="color:red">
+                    <b>Could not create GitHub Repo...</b><br />
```

Comment:
Revise to enhance UX. Users should be able to determine if GitHub repo should be created and they should be able to determine the name.
![image](https://user-images.githubusercontent.com/4316475/166116691-a9e44562-31a7-4c03-99b4-f56276cee1bc.png)